app/code/Magento/Eav/Test/Unit/Model/Entity/Attribute/Backend/JsonEncodedTest.php

```diff
@@ -94,5 +94,13 @@ public function testAfterLoad()
         );
         $this->model->afterLoad($product);
         $this->assertEquals([1, 2, 3], $product->getData('json_encoded'));
+
```

Comment:
Could you move this new code as separate method (test case)?

Reply:
@ihor-sviziev sure, done. Also I have squashed my commits.